randovania/games/samus_returns/logic_database/Area 3 - Factory Exterior.txt

```diff
@@ -855,12 +891,12 @@ Extra - asset_id: collision_camera_040
       All of the following:
           Morph Ball
           Any of the following:
-              Space Jump or Spider Ball
-              Bomb and Infinite Bomb Jump (Intermediate)
-              All of the following:
-                  # Destroying the Autrack
-                  Missile ≥ 20 or Power Bomb or Super Missile ≥ 4 or Screw Attack
-                  High Jump Boots or Unmorph Extend (Intermediate) or Wall Jump (Intermediate) or Simple IBJ
+              # Destroying the Autrack
+              Lightning Armor or Defeat Gold Autrack
```

Comment:
In other places in the DB, I added Lightning Armor to the Damage Boost groupings even though no damage is taken, since the same trick is at play. I'd regroup this.

Reply:
This hasn't been fixed and there's a duplicate lightning armor above

Reply:
fixed for real